lib/assortment.dart

```diff
@@ -107,6 +106,12 @@ class Assortment {
       event.preventDefault();
     }));
 
+    // prevent default browser handling for dropped data
```

Comment:
> In a web page, you should call the preventDefault() method of the event if you have accepted the drop so that the default browser handling does not handle the dropped data as well. For example, when a link is dragged to a web page, Firefox will open the link. By cancelling the event, this behavior will be prevented.

This is taken from the [Mozilla Developer Docs](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Drag_operations#drop).